source/features/ci-link.tsx

```diff
@@ -25,7 +25,7 @@ async function init(): Promise<false | void> {
 	}
 
 	// Append to title (aware of forks and private repos)
-	appendBefore('.pagehead h1', '.fork-flag', icon);
+	select('[itemprop="name"]')!.after(icon);
```

Comment:
This works, but it's slightly misaligned

<img width="344" alt="Screen Shot 2020-07-17 at 19 13 03" src="https://user-images.githubusercontent.com/1402241/87813247-d3dbc600-c861-11ea-863b-e2a134af3b2c.png">